citi-onboarding/Delta-odonto
# Feature/banner front

## WHAT I DID

Created the banner according to the mockup that can be found here : [https://www.notion.so/Mockup-fd77362ddd304287a73052e0e38eaa00#4f332ff59ba749c68774b58dab145a75](url) and connected it to the back-end

##HOW TO TEST

Follow the steps
1- Access this branch on your computer and type on the terminal theses commands:
`cd client`
`yarn start`

2- Then open another terminal at the same time and type in there theses commands:
`cd server`
`node index.js`

3- After this wait a few seconds and type on your browser: [localhost:3001/admin](url)

4- Click on the "banner" card and you will be able to see and change what have been written on the value proposition and its description

5- Later access this link: [http://localhost:3000](url) and you will be able to see a page with the banner. The same thing that was written on the value proposition and value proposition description is supposed to appear in the middle of the banner.
Beyond this is supposed to appear a link to "Nossos serviços" with a line under. If you see it from a mobile screen you will be able to see a logo at the top of the page and an arrow at the end of the banner.
